spec/jobs/digital_object_import_processing_job_spec.rb

```diff
@@ -150,21 +156,53 @@
       {}
     end
 
+    let(:digital_object_data_without_digital_object_type) do
+      {
+        'assign_uid' => 'new_uid'
+      }
+    end
+
+    let(:digital_object_data_with_conflicting_params) do
+      {
+        'uid' => digital_object.uid,
+        'assign_uid' => digital_object.uid,
+        'digital_object_type' => 'item'
+      }
+    end
+
     it 'finds an existing digital object when a uid is provided in the digital_object_data' do
       expect(
         described_class.digital_object_for_digital_object_data(existing_digital_object_data).new_record?
       ).to eq(false)
     end
 
-    it 'instantiates a new digital object of the expected type when a uid is NOT provided in the digital_object_data' do
```

Comment:
This is an important test that we don't want to remove.  The best way to think about this is:
- Creating a digital object without a uid is the default scenario.
- The `assign_uid` field can be optionally added to a csv row/digital_object_import, and that just allows a user to provide the specific uid that they want to be assigned to the brand new digital object before it's saved for the first time.